ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -186,6 +215,9 @@ public void after() {
    */
   @Test
   public void badInstantiationTest() throws Exception {
+    if (overrideEnabled) {
```

Comment:
use `assumeTrue(!overrideEnabled)` here.

Reply:
Good point. Take you suggestion, thanks!